README.md

```diff
@@ -445,3 +445,78 @@ their `CHANGELOG.md` file in the [integrations-core repository](https://github.c
 **Note for Chef Windows users**: as the datadog-agent binary available on the
 node is used by this resource, the chef-client must have read access to the
 `datadog.yaml` file.
+
+
+Development
+===========
+
+To contribute, you will have to follow the contribution guide in [CONTRIBUTING.md](./CONTRIBUTING.md).
+
+### Dependencies
+
+First, you should use [bundler](https://bundler.io) and the provided Gemfile to install the development and release dependencies.
+
+After having installed bundler, you can run the following command to install gem in a vendored folder:
+
+```bash
+bundle install --path .bundle
```

Comment:
CircleCI uses chefdk (that provides a virtual ruby environment like bundler), so this might mean that versions in the dev env are slightly different than in the CI env.

I think bundler is the go-to tool in the Ruby world, so I like advising to use `bundle` in the instructions, but maybe we should change the CI to also use it. What do you think? This is something we can do later.

I think the `--path .bundle` is the default and we can skip it.

Reply:
Yeah we should definitely use bundler so the tests are run reproducibly. 

The option `--path .bundle` is not the default one sadly. The default one is installing in the user wide bundle gems folder. Generally in `$HOME/.bundle/gems`